src/components/vm/disks/diskAdd.tsx

```diff
@@ -293,7 +390,7 @@ const CreateNewDisk = ({
                               validationFailed={validationFailed}
                               volumeName={volumeName}
                               idPrefix={idPrefix}
-                              onValueChanged={onValueChanged} />}
+                              onValueChanged={(key: keyof VolumeCreateBodyDialogValues, value) => onValueChanged(key, value)} />}
```

Comment:
Meh -- I figure that is necessary to appease the type checker, but it's unfortunate that this has a runtime impact.